### ISSUE TITLE
GraphQL Playground settings do not get set

### ISSUE THREAD

**Issue by omgrief** [apollographql/apollo-server#1746]
```js
const server = new ApolloServer({
  playground: {
    settings: {
      'request.credentials': 'include',
    }
  },
  ...otherServerSettings
);
```

Go to your GraphQL Playground route, and notice that the settings are just default settings.

related to github.com/prisma/graphql-playground/issues/741

Adding this here as there appears to be a copy of `graphql-playground-html` under the `@apollographql` namespace, and will possibly need to be updated when they fix this.

**Comment by capaj** [comment]
I have the same issue using the latest apollo-server-express 2.1.0

**Comment by nabeelz6** [comment]
I think they might have fixed this and related setting persistence issues in prisma/graphql-playground repo:
https://github.com/prisma/graphql-playground/pull/836

However, as @omgrief stated, there's a copy of playground under apollographql that should pick up the new 1.8.0 release?

**Comment by javlund** [comment]
Hi - I made a PR for it here, hope someone in charge will merge it: https://github.com/apollographql/graphql-playground/pull/1

**Comment by Binarytales** [comment]
I'm still having this issue. I have created a reduced test case here: https://github.com/Binarytales/jubilant-octo-telegram

I originally posted this issue against `graphql-playground` directly here: https://github.com/prisma/graphql-playground/issues/791#issuecomment-455272999

**Comment by Binarytales** [comment]
I've done a bit more digging and with the help of @Huvik I think that this issue wasn't actually fixed in `graphql-playground` till this PR https://github.com/prisma/graphql-playground/pull/836 was released in [version 1.8.0 ](https://github.com/prisma/graphql-playground/releases/tag/v1.8.0) (with [1.8.5](https://github.com/prisma/graphql-playground/releases/tag/v1.8.5) being that latest)

It looks like `apollo-server` is using `1.7.10`: https://github.com/apollographql/apollo-server/blob/master/packages/apollo-server-core/src/playground.ts#L12

**Comment by thomsbg** [comment]
I am still seeing this issue and have no workaround at the moment other than to wire up graphql-playground or graphiql manually. I am using `apollo-server-express@2.10.1` and `@apollographql/graphql-playground-html@1.6.24`. Not being able to set `'request.credentials': 'include'` means that this is unusable for my API which uses a session cookie for authentication.
